src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -55,7 +68,7 @@ public IConverterManager ConverterManager
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic")]
         public IBindingProvider AddTypeFilter(Func<Type, bool> predicate, IBindingProvider innerRule)
```

Comment:
I've commented on this in your other reviews - an IBindingProvider isn't a "rule" really. Hopefully we can change the naming across all this new stuff to match what the types/interfaces really are.